src/DocLayer.actor.cpp

```diff
@@ -855,6 +871,26 @@ int main(int argc, char** argv) {
 			}
 			break;
 		}
+#ifndef TLS_DISABLED
+		case TLSOptions::OPT_TLS_PLUGIN:
+			args.OptionArg();
```

Comment:
Is this unused code?

Reply:
It is. Just to be consistent with FDB. But I can remove it if it looks bad.

Reply:
To hide this option from the help text, I have to touch the `TLS_HELP` defined in KVS source.

Reply:
`fdbserver` only supports `OPT_TLS_PLUGIN` still as a hack for our testing harness so that it can always pass `--tls_plugin` regardless of fdbserver version.  If there's a moderately clean way to just never have this flag on the doc layer, that sounds better to me.